stl/CMakeLists.txt

```diff
@@ -244,6 +244,11 @@ set(IMPLIB_SOURCES
     ${CMAKE_CURRENT_LIST_DIR}/src/sharedmutex.cpp
     ${CMAKE_CURRENT_LIST_DIR}/src/syserror_import_lib.cpp
     ${CMAKE_CURRENT_LIST_DIR}/src/vector_algorithms.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/src/xonce2.cpp
+)
+set (IMPLIB_SOURCE_OBJECTS

```

Comment:
I think we want this to be added as imported object libraries.

https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#object-libraries
https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#imported-targets